boards/FSM/Test/Src/Test_FlowMeter.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Test_FlowMeter.h"
+
+DEFINE_FAKE_VALUE_FUNC(float, get_flow_rate);
+
+class FlowMeterTest : public FsmTest
```

Comment:
```suggestion
class FlowMeterTest : public testing::Test
```
So I am cleaning this up in https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/570 and long story short:

- `FsmTest` is to be removed
- If `StateMachineTest` ever wants `FlowMeterTest`, it will write `class StateMachineTest : public FlowMeterTest`

Also,
- In this case since no other Test modules is using FlowMeterTest directly, we should remove `Test_FlowMeter.h` entirely and just have `#include "Test_Fsm.h"` in `Test_FlowMeter.cpp`